src/tests/test_utils/common_test_utils/include/common_test_utils/all_close.hpp

```diff
@@ -128,10 +85,15 @@ namespace test {
 /// Returns true if shapes match and for all elements, |a_i-b_i| <= atol + rtol*|b_i|.
 template <typename T>
 ::testing::AssertionResult all_close(const ov::Tensor& a, const ov::Tensor& b, T rtol = 1e-5f, T atol = 1e-8f) {
-    auto a_t = std::make_shared<ngraph::runtime::HostTensor>(a.get_element_type(), a.get_shape(), a.data());
-    auto b_t = std::make_shared<ngraph::runtime::HostTensor>(b.get_element_type(), b.get_shape(), b.data());
+    std::vector<T> a_v(a.get_size());
+    ov::Tensor a_t(a.get_element_type(), a.get_shape(), static_cast<void *>(a_v.data()));
+    a.copy_to(a_t);
```

Comment:
Is it possible not to copy memory? Looks like vector may be created over `a.data()`

Reply:
If shapes are the same we can avoid copy